packages/react-bootstrap-table2/src/table-factory.js

```diff
@@ -19,8 +20,18 @@ export const wrapWithSort = props =>
 export const pureTable = props =>
   React.createElement(BootstrapTable, { ...props });
 
+export const wrapWithPagination = (props) => {
+  if (props.pagination) {
+    const { wrapper } = props.pagination;
+    const PaginationBase = wrapper(pureTable, Const);
+    return React.createElement(PaginationBase, { ...props });
```

Comment:
1. get pagination `wrapper` from props
2. call the wrapper factory function (`wrapper` is factory function`)